### PR TITLE
Upgrade OC packages and drivers

### PR DESCRIPTION
**Pull request by MikeAlhayek**
@Piedone @sarahelsaig

### CODE REVIEW

File: src/Modules/OrchardCore.Commerce.ContentFields/Settings/AddressFieldSettingsDriver.cs
Comment (sarahelsaig) [inline src/Modules/OrchardCore.Commerce.ContentFields/Settings/AddressFieldSettingsDriver.cs:14]: One arrow is after the line break, the other is before. This feels very random. Why?
Please move all `=>` at the end of the line instead of the beginning.
```suggestion
    public override IDisplayResult Edit(ContentPartFieldDefinition model, BuildEditorContext context) =>
        Initialize<AddressPartFieldSettings>("AddressFieldSettings_Edit", viewModel =>
```

File: src/Modules/OrchardCore.Commerce.ContentFields/Settings/PriceFieldSettingsDriver.cs
Comment (sarahelsaig) [inline src/Modules/OrchardCore.Commerce.ContentFields/Settings/PriceFieldSettingsDriver.cs:28]: :dizzy_face: 
```suggestion
    public override IDisplayResult Edit(ContentPartFieldDefinition model, BuildEditorContext context) =>
        Initialize($"{nameof(PriceFieldSettings)}_Edit", (Action<PriceFieldSettingsEditViewModel>)(viewModel =>
```

File: src/Modules/OrchardCore.Commerce.Payment.Stripe/Drivers/StripeApiSettingsDisplayDriver.cs
Comment (sarahelsaig) [inline src/Modules/OrchardCore.Commerce.Payment.Stripe/Drivers/StripeApiSettingsDisplayDriver.cs:46]: Doesn't need to be in a separate line. Please the same for all overrides of `SettingsGroupId`
```suggestion
    protected override string SettingsGroupId => GroupId;
```

File: src/Modules/OrchardCore.Commerce/Drivers/PriceDisplaySettingsDisplayDriver.cs
Comment (sarahelsaig) [inline src/Modules/OrchardCore.Commerce/Drivers/PriceDisplaySettingsDisplayDriver.cs:56]: You missed this one.
```suggestion
            .OnGroup(SettingsGroupId);
```

**Review by sarahelsaig — CHANGES_REQUESTED** [review]

**Comment by MikeAlhayek** [comment]
@sarahelsaig @Piedone hold off on merging this PR until we first merge https://github.com/OrchardCMS/OrchardCore/pull/16530 . Once merged, I'll fix any other warnings here before merging it

**Comment by MikeAlhayek** [comment]
@sarahelsaig @Piedone this PR is ready with all the changes from the drivers.

**Pull request merged**
